modules/deploy/pages/deployment-option/cloud/managed-connectors/transforms/smt-castredpanda.adoc

```diff
@@ -0,0 +1,27 @@
+= Cast (Redpanda)
```

Comment:
I have some doubts whether we should be telling about the Redpanda Cast. It was introduced for internal use and does not bring a lot comparing to the Cast SMT.